spotui/src/PlaylistMenu.py

```diff
@@ -39,9 +46,27 @@ def __select_playlist(self, playlist_name, playlist_id, playlist_uri):
         self.change_tracklist(self.api.get_playlist_tracks(playlist_id),
                               playlist_name, playlist_uri)
 
-    def __map_playlists(self, item):
-        available_space = self.endx - self.startx - 6
+    def __map_playlists(self, item, current_playlist_uri):
+        available_space = self.endx - self.startx - 3
+        
         item["text"] = truncate(item["text"], available_space)
+        
+
+        #injection of current playlist
+        #TODO: update this list when a new playlist is playing
+        #TODO: check context on new song and update panels as necessary
+        if str(item["uri"]) == current_playlist_uri:
+           
+            if len(item["text"].encode("utf-8")) < available_space:
+                item["text"] = pad_str(item["text"], available_space - 2)
+
+            item["text"] = "{0} ".format(item["text"])
+        else:    
+            #TODO: ljust does not account for bytes
+            item["text"] = pad_str(item["text"], available_space)
+        #end injection
```

Comment:
We store the results of get_playing() in MainForm so we don't need to be making the request here too.

Can we implement a public method call refresh_now_playing on this class (similar to what we have on TracksMenu) and call it inside our status_loop.

We don't even need to check at initial render here to be honest. Let's take this code out and just handle setting/unsetting the playlist indicator inside the new refresh_now_playing method.

At some point it would be nice to manage state a bit better so we're not constantly passing all this stuff around, but this is fine for now